inframanager/api_handler/api_handler.go

```diff
@@ -357,7 +357,7 @@ func InsertDefaultRule() {
 		log.Debugf("Entry %s %s %d already exists", macAddress, types.DefaultRoute, portID)
 	} else {
 		if ep.WriteToStore() != true {
-			err = fmt.Errorf("Failed to add mac: %s ip: %s port: %d  entry to the store",
+			log.Errorf("Failed to add mac: %s ip: %s port: %d  entry to the store",
```

Comment:
Didn't know if checking the error and returning was the right thing since the previous code wasn't returning from here. So converted to a log

Reply:
Yeah. It was intentional. We are not returning error for inserting default rule.